xlators/cluster/dht/src/dht-shared.c

```diff
@@ -493,9 +494,11 @@ dht_reconfigure(xlator_t *this, dict_t *options)
     }
 
     if (dict_get_str(options, "decommissioned-bricks", &temp_str) == 0) {
-        ret = dht_parse_decommissioned_bricks(this, conf, temp_str);
-        if (ret == -1)
-            goto out;
+        if (!(conf->decommission_in_progress)) {
```

Comment:
If we are trying to set the decommissioned bricks when we already have a list, shouldn't we return an error ?

Reply:
The scenario (which i admit is an edge case but still a bug) is:
1. decommission start: the option decommissioned-bricks is added to the vol file and being parsed by dht.
2. another configuration change (like setting a new loglevel): the decommissioned-bricks option still exists on the vol file.
3. the fix prevents another parse and counts on the fact that once a decommission is running it cannot be started again.

Reply:
This info is a good edition to the commit message IMO

Reply:
Added to scenario to the comment